priv/www/index.html

```diff
@@ -1,5 +1,6 @@
 <!doctype html>
 <meta http-equiv="X-UA-Compatible" content="IE=edge" />
```

Comment:
I wonder if this tag should be in the other file, or removed completely - https://stackoverflow.com/a/6771584

Reply:
this tag was already present and going through the SO post you linked, it seems it's only necessary for IE9 and versions below. What's your take?

Reply:
There are folks who would appreciate if we didn’t drop IE9 support at this point. Older versions can be safely ignored.